include/utils.h

```diff
@@ -13,6 +13,9 @@
 
 namespace newsboat {
 
+// Forward declaration for a type from htmlrenderer.h
+enum class LinkType;
```

Comment:
Nitpick: Might be nice to define in a separate header instead of `htmlrenderer.h` (e.g. `linktypes.h`) so these forward declarations are not needed.

Reply:
IMHO `LinkType` is logically a part of `HtmlRenderer`, so it shouldn't be in a separate header. I think I'll postpone this change until we have a third user for `LinkType` :)